.github/workflows/tagAndRelease.yml

```diff
@@ -98,7 +98,7 @@ jobs:
         with:
           email: ${{ secrets.IDEE_GH_EMAIL }}
       - run: |
-          git fetch
+          git fetch --unshallow
```

Comment:
@randi274 had some super googling to find the resolution to this issue.  Thread [here](https://salesforce-internal.slack.com/archives/G0213NY8PMH/p1676497532107069)